orttraining/orttraining/test/python/orttraining_test_ortmodule_api.py

```diff
@@ -269,20 +269,21 @@ def test_model_to_device_and_back_to_original(original_device, to_device):
     for _, parameter_value in model.named_parameters():
         assert parameter_value.device.type == original_device
 
-def test_model_with_different_devices_same_session():
-    N, D_in, H, D_out = 64, 784, 500, 10
-    model = NeuralNetSinglePositionalArgument(D_in, H, D_out)
-    model = ORTModule(model)
-
-    for i in range(5):
-        if i % 2 == 0:
-            device = 'cpu'
-        else:
-            device = 'cuda'
-
-        model.to(device)
-        x = torch.randn(N, D_in, device=device)
-        y = model(x)
+# @pytest.mark.skip(reason="TODO: ORTModule.to(device) is disabled for now")
```

Comment:
Doesn't this line already disable the test without having to comment the whole thing out?

Reply:
It was working on the dev machines, but somehow pytest.mark.skip is not working on the CI machine. 
Maybe due to old pytest version.